src/Service/DigitalPostHelper.php

```diff
@@ -72,6 +77,106 @@ public function sendDigitalPost(string $cpr, string $name, Address $address, str
         return $result;
     }
 
+    public function createDigitalPost(Document $document, string $subject, string $entityType, Uuid $entityId, array $digitalPostAttachments, array $digitalPostRecipients): void
+    {
+        $digitalPosts = [];
+
+        // Handle attachments and add extra digital post entities if necessary due to size or number of attachments
+        // Size limit for attachments and main document combined is 80mb
+        // The max number of attachments per digital post is 10
+        $documentPath = $this->getFilePath($document);
+        $currentSizeOfDigitalPost = $mainDocumentSize = filesize($documentPath);
```

Comment:
It may be worth letting the `DigitalPost` entity keep track of the total size of files in the post by adding a `getTotalFileSize()` function, say.

Reply:
Added `totalFileSize` property to `DigitalPost`, to be updated when adding document or attachments.